lib/DefinePlugin.js

```diff
@@ -29,7 +29,7 @@ class RuntimeValue {
 			}
 		}
 
-		return this.fn();
+		return this.fn(parser.state.module);
```

Comment:
```suggestion
		return this.fn({ module: parser.state.module });
```

We may need to add more info later...

Reply:
absolutely